### ISSUE TITLE
Clone from gitlab.com via ssh fails

### ISSUE THREAD

**Issue by nakatanakatana** [x-motemen/ghq#384]
Error occurs when `ghq get` on a repository that has not been cloned.

## reproduction

```
$ ghq get git@gitlab.com:gitlab-org/gitlab.git                                                                                                                                                                                                   clone ssh://git@gitlab.com/~/gitlab-org/gitlab.git -> /home/tanaka/repos/gitlab.com/~/gitlab-org/gitlab
     error failed to get "git@gitlab.com:gitlab-org/gitlab.git": unsupported VCS, url=ssh://git@gitlab.com/~/gitlab-org/gitlab.git: Get "ssh://git@gitlab.com/~/gitlab-org/gitlab.git?go-get=1": unsupported protocol scheme "ssh"

$ ghq --version
ghq version 1.6.0 (rev:7e11957)
```

## problem

Perhaps the following PRs that came in 1.6.0 are causing the problem.

https://github.com/x-motemen/ghq/pull/378/files

**Comment by Songmu** [comment]
Thanks for the report.
The pull request seems to have a large impact, so I guess we should revert it.

**Comment by Songmu** [comment]
I fixed and released it at #383. Please try it.
